src/CaptureFileInfo/ManagerTest.cpp

```diff
@@ -83,6 +84,38 @@ TEST(CaptureFileInfoManager, AddOrTouchCaptureFile) {
   manager.Clear();
 }
 
+TEST(CaptureFileInfoManager, AddOrTouchCaptureFilePathFormatting) {
+  QCoreApplication::setOrganizationName(kOrgName);
+  QCoreApplication::setApplicationName(
+      "CaptureFileInfo.Manager.AddOrTouchCaptureFilePathFormatting");
+
+  Manager manager;
+  manager.Clear();
+  ASSERT_TRUE(manager.GetCaptureFileInfos().empty());
+
+  // Currently AddOrTouchCaptureFile uses path operator== to discern whether 2 paths are the same.
+  // This test uses the same mechanism (via std::set), which makes this more of a smoke test.
+  // TODO(http://b/218298681) use std::filesystem::equivalent instead of operator== and improve this
+  // test.
```

Comment:
I'm not sure if that is desired. I believe the recent captures should only operate on file paths. If the user maps their hard drive or network drive to a different drive letter the file won't be found anymore. That's easy to understand. If they map their network drive twice with different drive letters, `equivalent` might refuse adding a path to the list of recent captures because the file with the other drive letter is already there. This behaviour changes depending on what drives are currently mapped, so I believe keeping it to paths makes it easier for the user to understand what happens.

Reply:
I did not think about this specific case and I would guess there might be other similar cases that I did not think about. I am going to merge the chagne as is, so the release process can continue and add your remark to the bug